### ISSUE TITLE
Imported keys renamed

### ISSUE THREAD

**Issue by f00bar12** [connectbot/connectbot#549]
In previous versions when I imported keys they took the name of the file the key was stored in. Now the keys are named with numbers. I have multiple keys and this makes using them more difficult to use, as opposed to the old behavior of maintaining the name of the file. I can't find a way to rename the imported keys, am missing something?

**Comment by f00bar12** [comment]
Yes I know I asked about this before, but as it was inside another issue I closed that and am asking under its own subject.

**Comment by LeCoyote** [comment]
Seconded. This is a separate problem from the DSA deprecation. Being unable to rename keys is quite a pain, especially as there is no way to predict/control how they will be renamed during import.

**Comment by f00bar12** [comment]
Apparently this was somewhat fixed.  Now if I import a key it is no longer renamed into some random number. The name is now in the format: "raw/:<path to the file/filename>". This is a large improvement to the previous behavior, but I would prefer if the path was omitted as the name of the key file itself is mostly pushed off the right side of the screen.  Ideally the name would be the filename by itself, and even better would be that you could set a friendly name for the key that is displayed instead of the path and filename.  

I do very much appreciate that currently I can differentiate the keys from each other as it makes the software so much more usable!  Hopefully a rename/friendly name option will be available to deal with this.

**Comment by f00bar12** [comment]
Got a new phone (Galaxy S9) and now it seems we're back to the old behavior of renaming keys with a number.  I don't see any way to edit the 'name' given to the keys or to set it during import.  This makes using multiple keys somewhat difficult, has anyone come up with a workaround or a fix?

**Comment by edmundlaugasson** [comment]
Having same issue and question.

**Comment by JsBergbau** [comment]
Issue still exists.

**Comment by scottwallacesh** [comment]
A couple of years on and this is still an issue.

**Comment by nindanaoto** [comment]
I couldn't reproduce the issue with the current main. It might be fixed already. The imported key's name is now the file name.
